nicole-Borgaard/hw_3/2-object-homework/app.js

```diff
@@ -0,0 +1,160 @@
+  // 'Car'
+  // Description: Our app manages car sales.
+  // Every 'car' has
+  // - a make
+  // - a model
+  // - a year
+  // Every 'car' can
+  // - drive
+  // - brake
+  // - park
+
+// Answer Starts Here
+
+//Things to track are events
+Every event has 
+ -  Due date & time 
+ -  Create Date & time
+ -  Description
+ -  Participants 
+ -  Tags
+ events can
+ - be created
+ - be edited
+ - be deleted
+
+Every day has
+- Events
+- Hours
+- Minutes
+every day can
+- have a new event
+- can be marked as a holiday
+- have an event deleted
+
+Every participant has
+- events
+- tags
+- available hours
+every participant can
+- Create events
+- edit events they cerated
+- delete events they created
+
+
+
+// Answer Ends Here
+
+
+// 2. Create an Object literal that lines up with the following description. Store it in the variable 'pet_owner', below. Be sure to give it reasonable values for each of its properties.
+
+  // 'Owner'
+  // Description: We are making an app for a veterinary clinic - it manages pets' vaccinations.
+  // Every 'owner' has:
+  // - a name
+  // - an address
+
+// Answer Starts Here
+
+var pet_owner = {
+    "owners":[
+      {
+        "name"      : "adam",
+        "address"  : "123 main street"
+      },
+      {
+        "name"      : "adam",
+        "address"  : "123 main street"
+      },
+      {
+        "name"      : "adam",
+        "address"  : "123 main street"
+      }
+    ]
+};
+
+
+// Answer Ends Here
+
+
+// 3. Create an Object literal that lines up with the following description. Store it in the variable `some_pet`, below.
+
+  // Pet
+  // Description: We are making an app for a veterinary clinic - it manages pets' vaccinations.
+  // Every 'pet' has:
+  // - a name
+  // - a species
+  // - a breed
+  // - a noise that it can make (e.g. 'bark', 'meow', etc.)
+  // Every pet can:
+  // - make noise (each pet makes its own unique noise, as specified by `noise`.
+
+// Answer Ends Here
+
+// Answer Starts Here
+var some_pet = {
+    "dog":[
+      {
+        "name"    : "frank",
+        "species" : "Canis lupus",
+        "breed"   : "Affenpinscher",
+        "noise"   : "bark"
+      },
+      {
+        "name"    : "damian",
+        "species" : "Canis lupus",
+        "breed"   : "Barbet",
+        "noise"   : "bark"
+      },
+      {
+        "name"    : "damian",
+        "species" : "Canis lupus",
+        "breed"   : "Havanese",
+        "noise"   : "bark"
+      }
+    ],
+    "cat":[
+      {
+        "name"    : "ann",
+        "species" : "Felis catus",
+        "breed"   : "Balinese",
+        "noise"   : "meow"
+      },
+      {
+        "name"    : "barb",
+        "species" : "Felis catus",
+        "breed"   : "Bengal",
+        "noise"   : "meow"
+      },
+      {
+        "name"    : "cat",
+        "species" : "Felis catus",
+        "breed"   : "Tonkinese",
+        "noise"   : "meow"
```

Comment:
This is... an oddly clean looking object... except it's in my folder and not yours. :P 
Unless you want to give me all your cats, dogs, and monkeys, go ahead and move anything you accidentally dropped into my folder back into yours, then git add, commit, and push back up. Your pull request will automatically update.